exercises/pascals-triangle/src/test/kotlin/PascalsTriangleTest.kt

```diff
@@ -1,123 +1,88 @@
 import org.junit.Test
 import org.junit.Ignore
-import org.junit.Rule
-import org.junit.rules.ExpectedException
 import kotlin.test.assertEquals
 
 class PascalsTriangleTest {
-
-    @Rule
-    @JvmField
-    var expectedException: ExpectedException = ExpectedException.none()
-
     @Test
-    fun zeroRows() {
-        val expectedOutput = emptyList<List<Int>>()
+    fun `zero rows`() = assertTriangleEquals(
+            0,
```

Comment:
nit: Hmm. Seems like IDEA using 8 symbols for continuation indentation. I don't remember if I changed it manually to 4 (to comply detekt/ktlint rules) or just imported official style guide.